crates/grit-core-patterns/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "grit-core-patterns"
```

Comment:
Yeah, I’ve asked myself this as well 😅

I don’t see this crate as a full engine, because it’s missing quite a bit compared to `marzano-core` or the Biome engine that will be built on top of this. The patterns are the heart of it, but it’s true it contains a bunch of implementations on top of the patterns, as well as auxiliary stuff like bindings and state and such as well.

Maybe `grit-pattern-engine`?

Reply:
Maybe `grit-pattern-matcher`?